third_party/move/move-compiler-v2/tests/checking/inlining/lambda.exp

```diff
@@ -43,7 +43,7 @@ module 0x42::LambdaTest2 {
           }
         }
     }
-    spec fun $foreach<T>(v: &vector<#0>,action: |&#0|());
+    spec fun $foreach<T>(v: &vector<#0>,action: |&#0|);
```

Comment:
Any chance we can show the type parameter `T` in these types here?

Reply:
In the debug output, the number is more useful IMO.